cves/CVE-2018-12460.yml

```diff
@@ -64,7 +65,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  When converting from a modified AVI file to an MPEG4 file, a logic check would
+  compare two fields of a studio profile that could potentially not be the same 
+  or even not instantiated. This could led to a NULL pointer dereference that could
+  crash the system, leading to a denial of service.
```

Comment:
Explaining what AVI and MPEG4 files are or do would help.

Reply:
added explanation to what an AVI file and MPEG4 file constitute